extensions/blocks/map/component.js

```diff
@@ -312,20 +325,22 @@ export class Map extends Component {
 			window.addEventListener( 'resize', this.debouncedSizeMap );
 		} );
 	}
-	googlePoint2Mapbox( google_point ) {
-		const mapCenter = [
-			google_point.longitude ? google_point.longitude : 0,
-			google_point.latitude ? google_point.latitude : 0,
-		];
-		return mapCenter;
-	}
+	googlePoint2Mapbox = google_point =>
+		google_point.hasOwnProperty( 'lat' ) && google_point.hasOwnProperty( 'lng' )
+			? google_point // Already a valid Mapbox point.
+			: {
+					// Legacy point, supported here to avoid block deprecation.
+					lng: google_point.longitude ? google_point.longitude : 0,
+					lat: google_point.latitude ? google_point.latitude : 0,
```

Comment:
I think we could simplify this with:
```js
{
  lat: google_point.latitude || 0,
  lng: google_point.longitude || 0,
}
```

Reply:
Much nicer: https://github.com/Automattic/jetpack/pull/14417/commits/736ea751f3bafb76e07c33fb9854eda623b45b5f